openwisp_controller/config/tests/test_device.py

```diff
@@ -322,3 +322,20 @@ def test_check_name_changed_query(self):
         with self.assertNumQueries(2):
             device.name = 'changed'
             device._check_name_changed()
+
+    def test_device_name_changed_emitted(self):
+        org = self._get_org()
+        device = self._create_device(name='test', organization=org)
+
+        with catch_signal(device_name_changed) as handler:
+            device.name = "newtestdevice"
```

Comment:
please use single quotes for consistency with the rest of the code base, applies to all other lines